mir_eval/hierarchy.py

```diff
@@ -98,6 +100,29 @@ def _hierarchy_bounds(intervals_hier):
     return min(boundaries), max(boundaries)
 
 
+def _align_intervals(int_hier, lab_hier, t_min=0.0, t_max=None):
+    '''Align a hierarchical annotation to span a fixed start and end time.
+
+    Parameters
+    ----------
+    int_hier : list of list of intervals
+    lab_hier : list of list of str
+        Hierarchical segment annotations, encoded as a
+        list of list of intervals (int_hier) and list of
+        list of strings (lab_hier)
+
+    Returns
+    -------
+    intervals_hier, labels_hier
```

Comment:
I would clearly specify the types of the returned values. I would also enumerate them separately.